### ISSUE TITLE
can't get the value of the selected date

### ISSUE THREAD

**Issue by claymore07** [javadbat/jb-date-input#14]
Hi,
I'm using this package on my Laravel blade form, It's work fine but when I submit the form, no value has been passed to server.
could you help me?

`<jb-date-input name="publish" label="تاریخ شمسی با اعداد فارسی" input-type="JALALI" use-persian-number message="کارکتر های عددی تایپ شده و تقویم به صورت فارسی خواهند بود">loading</jb-date-input>`

**Comment by javadbat** [comment]
currently, you can use JS to get the value of the date like this: `document.querySelector('jb-date-input').value` but in your case, you need to make a component form associated.
actually, I never use this component in such a way and I need to create a test environment and then make it compatible with it.

**Comment by claymore07** [comment]
Thanks a lot for your quick response,
I solved it by adding a hidden input
`<input type="hidden" name="published_at" id="published_at" value="">`

and adding event listener on the change of the date picker.

```
const datapicker = document.querySelector('jb-date-input');
    const publishedAt = document.getElementById('published_at');
    datapicker.addEventListener('change', function(){
        publishedAt.value = document.querySelector('jb-date-input').value;
    })
```

I am not a JS expert, but it solved my challenge.
I really liked your package design and implementation, working and finding a good package for Jalali date picker is tiring task. 

I've request, if it possible, add a time picker too.
Best regards

**Comment by javadbat** [comment]
first thank you you for your kind response. second for the time I have `jb-time-input` you can use and see this list for more component [list of jb component](https://www.npmjs.com/search?q=keywords:jb)
but as I research your case you better use this code to add date value to your form:
```
const form = document.querySelector('form');
// FormData event is sent on <form> submission, before transmission.
// The event has a formData property
form.addEventListener('formdata', ({formData}) => {
  // https://developer.mozilla.org/docs/Web/API/FormData
  formData.append('birthdate', document.querySelector('jb-date-input').value);
});
```
besides that, I'm working on components to make it fully compatible with form but it takes time

**Comment by claymore07** [comment]
I grateful for your help.

For the time picker, I mean, if it is possible, integrate it with your date picker. and a simple switch between Jalali and Georgian,
and these are absolutely a suggestion, I understand that you have busy schedule.

Wish you the best

**Comment by javadbat** [comment]
I just fix it can you update to v 3.8.0 and test it and tell me the result?
